common/djangoapps/embargo/tests/test_api.py

```diff
@@ -0,0 +1,399 @@
+"""
+Tests for EmbargoMiddleware
+"""
```

Comment:
Since these are tests for middleware (not the Python API in `api.py`), this file should probably be called `test_middleware.py`